src/lib/tvgCommon.h

```diff
@@ -65,8 +65,11 @@ using namespace tvg;
 enum class FileType { Tvg = 0, Svg, Raw, Png, Jpg, Unknown };
 
 #ifdef THORVG_LOG_ENABLED
-    #define TVGLOG(tag, fmt, ...) fprintf(stdout, tag ": " fmt "\n", ##__VA_ARGS__)  //Log Message for notifying user some useful info
-    #define TVGERR(tag, fmt, ...) fprintf(stderr, tag ": " fmt "\n", ##__VA_ARGS__)  //Error Message for us to fix it
+    constexpr auto ErrorColor = "\033[31m";  //red
+    constexpr auto LogColor = "\033[32m";    //green
+    constexpr auto ResetColor = "\033[0m";   //default
+    #define TVGERR(tag, fmt, ...) fprintf(stderr, "%s" tag "(%s l.%d): " fmt "%s\n", ErrorColor, __FILE__, __LINE__, ##__VA_ARGS__, ResetColor)
+    #define TVGLOG(tag, fmt, ...) fprintf(stderr, "%s" tag ": " fmt "%s\n", LogColor, ##__VA_ARGS__, ResetColor)
```

Comment:
I think TVGLOG also needs FILE, LINE. any reason TVGLOG doen't contain them?

Reply:
because I thought that log is more for the user then for us, and I'm not sure how important for a user is to see the file&line. the msg is less readable in my opinion:
```SVG (../src/loaders/svg/tvgXmlParser.cpp l.523): Unsupported attributes used [Elements type: Path][Id : path4657][Attribute: enable-background][Value: accumulate]```
vs
```SVG: Unsupported attributes used [Elements type: Path][Id : path4657][Attribute: enable-background][Value: accumulate]```

but I consulted this and apparently people prefer to see file&line :D so I changed it